### ISSUE TITLE
Modjoined battles send |noinit|nonexistent|, client shows "battle expired"

### ISSUE THREAD

**Issue by Slayer95** [Zarel/Pokemon-Showdown#5676]
- We probably don't need to lie about the existence of battle rooms.
- Causes the client to GET the corresponding replay data needlessly and fail with confusing message.

**Comment by Zarel** [comment]
Hm, I don't know how to feel about this.

I think it probably isn't a leak to send "Access Denied" instead of "nonexistent" for every modjoined room, but let's ask around first.

**Comment by Slayer95** [comment]
Note that as of now, modjoined battle rooms seem to be reported in ``/cmd userdetails``. So something should be done either way.

**Comment by Slayer95** [comment]
AFAICT, this is unneeded after #5678 .

Closing in favor of the status quo.

**Comment by Zarel** [comment]
I actually would prefer to change the status quo here. "Privacy over usability" seems like the wrong tradeoff, especially since the privacy gains are minimal.
